src/program/snabbnfv/nfvconfig.lua

```diff
@@ -23,27 +23,29 @@ function load (file, pciaddr, sockpath)
       print(format("could not find device information for PCI address %s", pciaddr))
       main.exit(1)
    end
+   local driver = require(device_info.driver).driver
 
    local ports = lib.load_conf(file)
    local c = config.new()
    for _,t in ipairs(ports) do
-      local vlan, mac_address = t.vlan, t.mac_address
+      local vlan, mac_address, vmdq = t.vlan, t.mac_address, t.vmdq
+      local disable_mrg_rxbuf = t.disable_mergeable_rx_buffer
       local name = port_name(t)
       local NIC = name.."_NIC"
       local Virtio = name.."_Virtio"
-      local vmdq = true
+      if vmdq == nil then vmdq = true end
```

Comment:
I don't think that will work. VMDq is the hardware feature that received packets to be dispatched across different queues based on their destination (DMAC w/ optional VLAN). I think it only makes sense to disable VMDq if you have one VM on the port that is supposed to receive all traffic. Having said that, the `intel10g` module API is not a 1:1 mapping onto the hardware so it is possible that the `vmdq=true` parameter has a more peculiar meaning that this.